src/kata4.humanCatDogYears.js

```diff
@@ -1,3 +1,11 @@
-const humanCatDogYears = (number) => {};
+const humanCatDogYears = (number) => {
+const catYears = ([number-2]* 4) + 24;
+const dogYears = ([number-2]* 5) + 24;
+const humanCatDogYears = [number, catYears, dogYears]
+
+return number;
+};
+
```

Comment:
Theres a couple of things to talk about here. At first glance everything seems to be fine as the tests are passing, so therefore the function must be working. But if you were to change the second or third numbers in the tests, this will now fail. So whats happening here?

The test is matching the first number returned by the function. Looking at your code you are only returning the original number passed into the function, so the code on lines 2-5 is never used. We can fix this by returning the array we are supposed to, the one you've already made on line 4:
```
const humanCatDogYears = (number) => {
  const catYears = [number - 2] * 4 + 24;
  const dogYears = [number - 2] * 5 + 24;
  return [number, catYears, dogYears];
};
```
However, your test will still fail. This is because of the value of what is returned, in the test you are looking for 3 values like this:
```
expect(humanCatDogYears(10)).toEqual(10, 56, 64);
```
But in the function you are returning an _array_:
```
[10, 56, 64]
```
So you need to update your test to include the array brackets.